vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -41,6 +42,20 @@ def ems_inv_to_hashes
 
       private
 
+      def all_server_resources
+        return @all_server_resources if @all_server_resources
+
+        resources = []
+        stacks.each do |stack|
+          # Nested depth 50 just for sure, although nobody should nest templates that much
```

Comment:
Looks like this comment is no longer necessary (here, at least) ... now that the logic was refactored into the common code for parsing.